.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Integration (CI)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to the repository
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

Comment:
Probably right now it's not a big deal but if we can give it a try using Node v18, it would be nice since it's the active LTS version.

![Screen Shot 2022-10-31 at 09 28 20](https://user-images.githubusercontent.com/17997480/199007784-f5f1a5e1-40d3-4409-87d5-876857bdbf82.png)

Reply:
Actually, v16 is the one the action uses by default
https://github.blog/changelog/2021-12-10-github-actions-github-hosted-runners-now-run-node-js-16-by-default/

Reply:
Ok! So I'm setting node version to 18